public/.gitignore

```diff
@@ -0,0 +1,2 @@
+assets/
```

Comment:
Remove file and add code in the `.gitignore` file at the project root. Why ?

When I have finished the symfony recipes for the `bolt/core` component, installing with composer will directly configure the symfony project and add things to the project's `.gitignore` file and not to the public folder.

And the good practice with the `.gitignore` files in mono project is to have only one that drives the project (what we do with symfony when installing components via flex and its recipes).

On the other hand in a monolithic project, which is not the case here, is to have one file per project because they are often different technologies. [For example the project api-platform](https://github.com/api-platform/api-platform)